### PR TITLE
De Minimis Endpoint

### PR DESCRIPTION
**Pull request by tmhammer**

### CODE REVIEW

**Comment by calumhalcrow** [comment]
@tmhammer couple of problems:

- Explorer pulls documents from staging (api.govwizely.com), due to the fact that it's used to demo new endpoints (which typically only exist on staging at time of demo). De Minimis isn't available on staging, so Explorer cannot find the endpoint.
- Even if it was on staging, the endpoint would be /de_minimis, not /de_minimis_rates.

File: config/environment.js [hunk 1]
```diff
@@ -5,7 +5,8 @@ module.exports = function(environment) {
     environment: environment,
     locationType: 'auto',
     webservicesBaseURL: 'http://localhost:3000',
-    webservicesApiKey: 'cfVhA_8HepggR81yU6yo1KGN',
+    //webservicesApiKey: 'cfVhA_8HepggR81yU6yo1KGN',
+    webservicesApiKey: 'XWZCgKOL8xefphX8Q9DtYY4u',
```
Comment (calumhalcrow) [inline config/environment.js]: Just curious: why are you doing this?

**Comment by tmhammer** [comment]
Thanks @calumhalcrow, that is very strange.  Yesterday I know there was an endpoint called "de_minimis_rates" on staging.  I will get in touch with Stuart about that and will block this for now.

And I'll fix the api_key thing, I just did that to match the local user I have and forgot to change it back.

**Comment by tmhammer** [comment]
This should be good to go now @calumhalcrow.  Story:  https://www.pivotaltracker.com/story/show/107737580  Already on staging:  http://govwizely.github.io/explorer/#/de-minimis

**Pull request merged**